cargo-dist-schema/src/snapshots/cargo_dist_schema__emit.snap

```diff
@@ -1,16 +1,20 @@
 ---
 source: cargo-dist-schema/src/lib.rs
-expression: "serde_json::to_string_pretty(&schema).unwrap()"
+expression: json_schema
 ---
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "DistManifest",
   "description": "The final report of cargo-dist",
   "type": "object",
-  "required": [
-    "releases"
-  ],
   "properties": {
+    "dist_version": {
+      "description": "The version of cargo-dist that generated this",
+      "type": [
+        "string",
+        "null"
```

Comment:
It can't actually be in practice, but the schema generator can't quite tell (because we skip_deserializing_if Option::is_none, but that doesn't mean a hand-crafted version couldn't manually put a null there). See also the above comment about bootstrapping.